src/main/java/Util.java

```diff
@@ -10,12 +10,11 @@
     static int height = 600;
     static int spaceBtwRows = 15;
     static int sidePadding = 10;
-    static int drawingBoxLeftPadding = 410;
+    static int drawingBoxLeftPadding = 320;
```

Comment:
Why did you change these values?

Reply:
Because after making the score value bold, "Number of spots" was not fitting in the screen. 
You can try using 410 , you'll understand better what I'm saying